tests/db/model/collectorGroup/utils.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * tests/db/model/collectorGroup/utils.js
+ */
+
+'use strict'; // eslint-disable-line strict
+const tu = require('../../../testUtils');
+const testStartTime = new Date();
+const collName = `${tu.namePrefix}Collector`;
+const collGroupName = `${tu.namePrefix}collGroupName`;
+const collector = {
+  name: collName,
+  description: 'This is a mock collector object for testing.',
+  helpEmail: 'test@test.com',
+  helpUrl: 'http://test.com',
+  host: 'xxx-yyy-zzz.aaa.bbb.ccc.com',
+  ipAddress: '123.456.789.012',
+  osInfo: {
+    hostname: 'testHostname',
+    username: 'testUsername',
+  },
+  processInfo: {
+    execPath: 'testExecPath',
+    memoryUsage: {
+      heapTotal: 1234,
+      external: 5678,
+    },
+    version: 'v1',
+    versions: { a: 'a', b: 'b' },
+  },
+  version: '1.0.0',
+};
+
+const standardCollectorGroup = {
+  name: collGroupName,
+  description: 'This is a dummy collectorGroup object for testing.',
+};
+
+module.exports = {
+
+  getCollectorObj() {
+    return JSON.parse(JSON.stringify(collector));
+  },
+
+  getCollectorGroupObj() {
```

Comment:
I just use it to differentiate that it is a raw object and not something returned from Sequelize (like a db object) - I am open to better suggestions :)

Reply:
I'd call just createCollectorGroup... 1 - get has a different meaning and in that case, you are creating something; 2 - Would remove Obj since there is no distinction in the test with nameDB and nameObj - I feel a bit confused about the name convention, looking something as Hungarian notation.